RequestService/RequestService.Handlers/GetRequestsByFilterHandler.cs

```diff
@@ -49,6 +97,31 @@ public async Task<GetRequestsByFilterResponse> Handle(GetRequestsByFilterRequest
 
             var requestSummaries = _repository.GetRequestsByFilter(request, referringGroups);
 
+            if (requestSummaries.Count>0)
```

Comment:
Not sure whether this logic is in the correct place...  We need to:
1. Find all jobs passing the filters (including ExcludeSiblingsOfJobsAllocatedToUserID)
2. Expand out to the request level
3. Return all jobs in those requests

Suppose a request has 4 jobs (2 dates, 2 volunteers required on each): 1a, 1b, 2a, 2b.

* If I have accepted 1a then 1b would be excluded at step 1 above.
* However, if I have accepted neither 2a nor 2b then those would pass step 1.
* So, step 3 would return all four jobs.